### PR TITLE
checks: allow for multi-line code blocks

### PR DESCRIPTION
**Pull request by jorgeorpinel**
per https://github.com/iterative/dvc.org/pull/2054#pullrequestreview-558337058

> You may disregard these recommendations if you used the **Edit on GitHub** button from dvc.org to improve a doc in place.

❗ Please read the guidelines in the [Contributing to the Documentation](https://dvc.org/doc/user-guide/contributing/docs) list if you make any substantial changes to the documentation or JS engine.

🐛 Please make sure to mention `Fix #issue` (if applicable) in the description of the PR. This causes GitHub to close it automatically when the PR is merged.

Please choose to [allow us](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to edit your branch when creating the PR.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

### CODE REVIEW

**Comment by jorgeorpinel** [comment]
I ran `yarn format-all` after the prettier config change and this is what happened. Waiting for test app deployment to double check the rendered results... ⌛

File: content/blog/2020-04-06-april-20-dvc-heartbeat.md [hunk 1]
```diff
@@ -14,8 +14,7 @@ descriptionLong: |
   projects by our users and big ideas about best practices in ML and data
   science.
 picture: 2020-04-06/april_header.png
-pictureComment:
-  A view from [Barrancas del
+pictureComment: A view from [Barrancas del
   Cobre](https://en.wikipedia.org/wiki/Copper_Canyon), shot by Jorge Orpinel
```
Comment (jorgeorpinel) [inline content/blog/2020-04-06-april-20-dvc-heartbeat.md:18]: BTW I guess this should be manually reformatted so the link isn't broken into 2 lines. Let's see if it lets me...

**Comment by shcheklein** [comment]
I still don't quite understand what does this change exactly mean

File: content/blog/2020-04-06-april-20-dvc-heartbeat.md (hunk 1 above)
Comment (jorgeorpinel) [reply content/blog/2020-04-06-april-20-dvc-heartbeat.md:18]: It does 🙂 (3a27d70a)

**Comment by jorgeorpinel** [comment]
@shcheklein per https://prettier.io/docs/en/options.html#prose-wrap this means to "wrap markdown text as-is" (which BTW is the default behavior). So we should rather see why it was `"always"` before (which means "wrap if it exceeds the print width"). I guess it's about auto-breaking long lines... So perhaps you're right that this might have big implications...

File: .prettierrc [hunk 2]
```diff
@@ -6,5 +6,4 @@
   "printWidth": 80,
   "tabWidth": 2,
   "useTabs": false,
-  "proseWrap": "always"
```
Comment (jorgeorpinel) [inline .prettierrc:9]: (`"preserve"` is the default value anyway.)

File: content/docs/user-guide/dvc-files.md [hunk 3]
```diff
@@ -143,7 +143,7 @@ stages:
     metrics:
       - performance.json
   training:
-    desc: Training stage description
+    desc: Training stage description blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah
```
Comment (jorgeorpinel) [inline content/docs/user-guide/dvc-files.md:146]: Long line in code block

File: content/docs/user-guide/dvc-files.md [hunk 4]
```diff
@@ -161,7 +161,7 @@ stages:
 
 `dvc.yaml` files consists of a group of `stages` with names provided explicitly
 by the user with the `--name` (`-n`) option of `dvc run`. Each stage can contain
-the following fields:
+the following fields blah blah blah blah blah blah blah blah blah blah blah blah blah blah blah:
```
Comment (jorgeorpinel) [inline content/docs/user-guide/dvc-files.md:164]: Long line in md

**Review by jorgeorpinel — COMMENTED** [review]
Yeah with this change regular long lines in md don't get wrapped 🤦

**Comment by jorgeorpinel** [comment]
Ivan was right!

**Pull request closed without merging**